app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -459,6 +528,13 @@
     miq_tabs_show_hide("#smartstate_docker_tab", false);
     miq_tabs_init('#auth_tabs');
     $('#auth_tabs').show();
+%div{"ng-if" => "#{ng_model}.emstype === 'kubevirt'"}
+  :javascript
+    miq_tabs_show_hide("#amqp_tab", false);
+    miq_tabs_show_hide("#metrics_tab", false);
+    miq_tabs_show_hide("#ssh_keypair_tab", false);
+    miq_tabs_init('#auth_tabs');
+    $('#auth_tabs').show();
```

Comment:
I think we need similar adjustments here in order to remove the gap between the 2 tabs.

As seen below in the screenshot, it looks like we have a partially invisible 'alerts' tab in the middle, which needs to be displayed only when `emsCommonModel.alerts_selection !== 'disabled'`

<img width="855" alt="screen shot 2018-01-17 at 12 05 06 pm" src="https://user-images.githubusercontent.com/1538216/35064390-b85e14ee-fb7e-11e7-9805-7e635f4efbb8.png">


And probably something similar for the 'metrics' tab as well.

Reply:
@AparnaKarve that's actually not the place where the semi-invisible tabls are show.
This section is for the 'Edit' kubevirt provider screen (we've disabled the 'Add' option)

![selection_184](https://user-images.githubusercontent.com/316242/35098098-8b6997a0-fc5b-11e7-9567-437be9110ecd.png)

The problematic part is https://github.com/ManageIQ/manageiq-ui-classic/pull/3143/files#diff-19776d82dcdf8bc1080fe79eb95a254bR562
The tabs that are being added under the ems_container.
I've updated the PR to hide them when they aren't used.